main.c

```diff
@@ -116,10 +117,21 @@
 #define TIMER_1KHZ		TIMER_5
 #define TIMER_IR_COMM	TIMER_1 // Timer number hardcoded into prox. processing code.
 
+#define MMA7660 0
+#define LIS2DE12 1
+   
+static int acc_type;
 static void acc_cb(int x, int y, int z, int tap) {
-	vmVariables.acc[0] = x;
-	vmVariables.acc[1] = y;
-	vmVariables.acc[2] = z;	
+	if (acc_type == MMA7660){
+		vmVariables.acc[0] = x;
+		vmVariables.acc[1] = y;
+		vmVariables.acc[2] = z;
+	}else{
+		vmVariables.acc[0] = x;//in FIFO no more division
+		vmVariables.acc[1] = y;
+		vmVariables.acc[2] = z;
+	}
+	
```

Comment:
What's the point of the if/else?

Reply:
at first implementation without the fifo buffer the value where 3 times bigger and I was making a division here. Then implementation changes and I manage to divid by 3 in the filter. Probably better to clean